content/TrueCommand/TCGettingStarted/InitialConfig.md

```diff
@@ -12,130 +12,158 @@ tags:
+2. [Create user accounts]({{< relref "/TrueCommand/TCGettingStarted/UserAccounts.md" >}}).
+3. Become familiar with the interface toolbars and icons. Review the [Interface Overview]({{< relref "/TrueCommand/TCGettingStarted/InterfaceOverview.md" >}}) article.
+4. [Connect your first TrueNAS system]({{< relref "/TrueCommand/TCGettingStarted/ConnectingTrueNAS.md" >}}).
 
-1. Create the administrator account. See [Creating the Admin Account]({{< relref "/TrueCommand/TCGettingStarted/AdminAccounts.md" >}}) for more information.
-2. Create the necessary user accounts. See [Creating User Accounts]({{< relref "/TrueCommand/TCGettingStarted/UserAccounts.md" >}}).
-3. Become familiar with the interface toolbars and icons. Review them here [Interface Overview]({{< relref "/TrueCommand/TCGettingStarted/InterfaceOverview.md" >}}).
-4. Connect your first TrueNAS system. See [Connecting Your First TrueNAS System]({{< relref "/TrueCommand/TCGettingStarted/ConnectingTrueNAS.md" >}}) for more information.
+## TrueCommand Administration Overview
+
+The **Administration** screen, available to users with administrator permissions, displays additional system details and offers a variety of TrueCommand configuration options through the system widgets or information cards.
 
-The **Administration** page, available to users with administrator permissions, displays additional system details and offers a variety of TrueCommand configuration options.
-Click the **Configure** <i class="material-icons" aria-hidden="true" title="Settings">settings</i> icon and select **Administration** to access the **Administration** screen. It is organized into **About**, **Certificates**, and **Configuration** widgets (functional and information cards).
+To access the **Administration** screen, click the **Configure** <i class="material-icons" aria-hidden="true" title="Settings">settings</i> icon and select **Administration**. 
 
 ![TrueCommand231AdminSystemInfo](/images/TrueCommand/2.3.1/TrueCommand231AdminSystemInfo.png "Administration Configuration")
 
-## TrueCommand Administration Overview
+The screen has seven widgets:
+* **[About](#about-widget)** that covers TrueCommand system information, iXsystem Support Contact information, and TrueCommand licenses.
+* **[Certificates](#certificates-widget)** where you can access system certificates, and upload or reset certificates.
+* **[Certificate Authorities](#certificate-authorities-widget)** where you can access or import certificate authorities.
+* **[Configuration](#configuration-widget)** where you configure general, ssl, and alert options, LDAP and SAML service, and Telemetry.
+* **[LDAP Servers](#ldap-servers-widget)** where you configure LDAP server settings.
+* **[LDAP Teams](#ldap-teams-widget)** where you can join LDAP teams.
+* **[Alert Recipients](#alert-recipients-widget)** where you configure users, and their email addreses, that receive system alerts.
```

Comment:
```suggestion
* **[Alert Recipients](#alert-recipients-widget)** where you configure users and their email addresses that receive system alerts.
```

Reply:
I'm surprised spellcheck didn't catch this. I need to use Hemmingway before I submit.